src/components/LineContent/index.tsx

```diff
@@ -53,11 +53,15 @@ export default class LineContent extends Component<any, any> {
         }
 
         return (
-            <span className={lineContent} style={style}>
-                {data &&
-                    data.map((part, n) => (
-                        <LinePart part={part} format={formatPart} key={`line-${number}-${n}`} />
-                    ))}
+            <span className={lineContent} style={style} onMouseUp={onMouseUp}>
+                {data?.map((part, n) => (
+                    <LinePart
+                        part={part}
+                        format={formatPart}
+                        key={`line-${number}-${n}`}
+                        style={{ width: "100%", display: "inline-block" }}
```

Comment:
Looks like you are still hardcoding this instead of using the prop.

Reply:
Bump.

Reply:
⭕